### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/config/

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#578] [Announcements]
# https://theme-hope.vuejs.press/zh/config/

配置介绍 查看详情 (intro.md); 多语言支持 查看详情 (i18n.md); 主题配置 主题基本配置 (theme/basic.md); 主题功能配置 (theme/feature.md); 主题布局配置 (theme/layout.md); 主题外观配置 (theme/appearance.md); 插件配置 主题插件介绍 (plugins...

https://theme-hope.vuejs.press/zh/config/

**Comment by yuanmoe** [comment]
好耶😆

**Comment by vipdcd** [comment]
请问该主题在一些浏览器中无法正常显示需要怎么处理，特别是侧边栏的链接无响应，比如当页面在WPS浏览器？

**Reply by Mister-Hope** [reply]
可能是一些新语法你所使用的浏览器没有支持到。我们原则上会兼容近两年的浏览器。
你可以试着调出相关浏览器的开发者工具提供错误栈，或者是准确提供他们使用的内核，注意，Chrome79之前的内核我们无计划支持。如果某些浏览器仍在使用小于该版本的内核，建议你像它们进行反馈内核过老的问题。

**Comment by icecome** [comment]
部署完成后，在文章的标头位置，存在一个路径信息，这是您的网站此位置的显示，并没有多余的文件夹路径信息。
![样式](https://pan.icecome.com/s/addto0)
<br>
这是我的网站的显示，存在多余的文件夹路径信息。
![样式](https://pan.icecome.com/s/n3xzdm)
<br>
希望您能帮助我解决此问题。

**Reply by icecome** [reply]
亟待解决，望拨冗答疑，非常感谢！
